src/tools/pubsub/pubsub.c

```diff
@@ -280,7 +280,7 @@ static char *expand_envvars(const char *src0);
 // FIXME: This is the same as the expand function in util. Merge.
 static char *expand_env(const char *name, char op, const char *alt) {
     char *env = NULL;
-    ddsrt_getenv(name, &env);
+    (void) ddsrt_getenv(name, &env);
```

Comment:
Given how ``ddsrt_getenv`` works and the way ``name`` is set, ignoring the return value is certainly acceptable in principle. But with the FIXME just a few lines above it, I wonder why this solution was chosen. I would prefer to see the function removed altogether and the one from ``ddsrt`` used instead.

Reply:
As far as I can see the call to `expand_envvars` can be replaced by a call to `ddsrt_expand_envvars_sh`. All the local environment functions in pubsub.c can then be removed.